e2e/wdio/platform/tests/link.e2e-spec.ts

```diff
@@ -43,7 +43,7 @@ describe('Link component test suite', function() {
                 if (i !== 8) {
                     expect(webDriver.getAttributeByName(linkPage.standardLinks, 'aria-label', i)).toBe(standardLinksAltTextArray[i]);
                     checkLinkData(linkPage.standardLinks, i);
-                    expect(webDriver.isElementClickeble(linkPage.standardLinks, 5000, i)).toBe(true);
+                    expect(webDriver.isElementClickable(linkPage.standardLinks, 5000, i)).toBe(true);
```

Comment:
This assertion won't work, since default time is the third argument in 'isElementClickable' method.

expect(webDriver.isElementClickable(linkPage.standardLinks, i)).toBe(true);

Reply:
you have a different version, not yet merged to base)
![2020-12-07_1](https://user-images.githubusercontent.com/47522152/101338741-0f654400-3886-11eb-801d-1fb73192d4df.png)

Reply:
actually I can see I can remove this wait time anyway here. Ok, will update